get-cli.sh

```diff
@@ -6,30 +6,30 @@ cli_version=${2:-latest}
 case ${option} in
```

Comment:
Aren't we going to update the package-base image version in Dockerfile as well?

Reply:
We haven't changed anything that relates to the Docker image, so I don't think we need to update the image version in the Dockerfile?

Thanks, yeah, I missed the other files, will make the updates.

Reply:
@nour-borgi I think I got them all, can you please just re-review this PR when you have a chance?